api-metastore/src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -72,10 +75,14 @@ public SubscriptionController(final SubscriptionService subscriptionService) {
     @RequestMapping(value = "/{id}/stats", method = RequestMethod.GET)
     public ItemsWrapper<SubscriptionEventTypeStats> getSubscriptionStats(
             @PathVariable("id") final String subscriptionId,
-            @RequestParam(value = "show_time_lag", required = false, defaultValue = "false") final boolean showTimeLag)
+            @RequestParam(value = "show_time_lag", required = false, defaultValue = "false") final boolean showTimeLag,
+            final HttpServletRequest request)
             throws InconsistentStateException,
             NoSuchEventTypeException, NoSuchSubscriptionException, ServiceTemporarilyUnavailableException {
+        final Span statsSpan = TracingService.extractSpan(request, "fetch_stats")
+                .setTag("subscription_id", subscriptionId)
+                .setTag("time_lag", showTimeLag);
```

Comment:
I think that `time_lag` should be renamed back to `show_time_lag`

Reply:
Fixed in https://github.com/zalando/nakadi/pull/1196/commits/2ee5b06be97b35973039e968b4dfb8246654e0d8